voxblox/src/utils/camera_model.cc

```diff
@@ -185,4 +184,57 @@ bool CameraModel::isPointInView(const Point& point) const {
   return true;
 }
 
+void CameraModel::getBoundingLines(AlignedVector<Point>* lines) const {
+  lines->clear();
+  lines->reserve(24);
+
+  // Transform the points again... This is just for visualization so we can
+  // waste a bit more time here.
+  AlignedVector<Point> transformed_corners;
+  transformed_corners.resize(untransformed_corners_.size());
```

Comment:
would it make sense to stick to the frame naming convention, i.e. corners_G and corners_C or something similar?

Reply:
Great idea, swapping the whole file to corners_G and corners_C. :) Much clearer I think!